Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/BaseClasses/PartialWheel.mo

```diff
@@ -0,0 +1,231 @@
+    "Supply air";
+  replaceable package Medium2 =
+    Modelica.Media.Interfaces.PartialCondensingGases
+    "Exhaust air";
+  parameter Modelica.Units.SI.MassFlowRate m1_flow_nominal
+    "Nominal supply air mass flow rate"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate m2_flow_nominal
+    "Nominal exhaust air mass flow rate"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.PressureDifference dp1_nominal(displayUnit="Pa") = 500
+    "Nominal supply air pressure drop"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.PressureDifference dp2_nominal(displayUnit="Pa") = dp1_nominal
+    "Nominal exhaust air pressure drop"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real P_nominal(final unit="W")
+    "Power consumption at the design condition"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsSenCoo_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsLatCoo_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsSenHea_nominal(
+    final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.Efficiency epsLatHea_nominal(
+    final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Nominal condition"));
+
+  parameter Modelica.Units.SI.Efficiency epsSenCooPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsLatCooPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the cooling mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsSenHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) sensible heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Part load effectiveness"));
+  parameter Modelica.Units.SI.Efficiency epsLatHeaPL(
+    final max=1) = 0.75
+    "Part load (75%) latent heat exchanger effectiveness at the heating mode"
+    annotation (Dialog(group="Part load effectiveness"));
+
+  Modelica.Blocks.Interfaces.RealOutput P(
+    final unit="W")
+    "Electric power consumption"
+    annotation (Placement(transformation(extent={{100,-10},{120,10}})));
+  Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.HeatExchangerWithInputEffectiveness hex(
```

Comment:
These should be protected -- no need to expose all these result variables.
But then, at the top-level, add a variable `epsLat` and `epsSen` to see what the actual efficiency is.